.github/workflows/ci.yml

```diff
@@ -33,6 +33,9 @@ jobs:
       - name: Build
         run: cargo build --all-targets --all-features --verbose
 
+      - name: Build `no-std`
```

Comment:
would be nice to mirror this in the `justfile` as well :)

https://github.com/ralexstokes/ssz-rs/blob/main/justfile#L9